scripts/disable-datastore-viewer.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Disable the datastore viewer, reload nginx and configure the firewall.
+usage() {
+        echo
+        echo "Usage: $0 <APP-ID>"
+        echo
+        echo "Disable the datatore viewer for the given application."
+        echo
+        echo "Options:"
+        echo "     -h        this message"
+        echo
+}
+
+# Parse command line arguments
+APP_ID=""
+while [ $# -gt 0 ]; do
+        if [ "$1" = "-h" -o "$1" = "-help" -o "$1" = "--help" ]; then
+                usage
+                exit 1
+        fi
+        if [ -n "$1" ]; then
+                APP_ID=$1
+                shift
+                continue
+        fi
+done
+if [ ! -n "$APP_ID" ]; then
+	usage
+	exit 1
+fi
+
+# Sanity checks.
+if [ ! -e /etc/nginx/sites-enabled ]; then
+        echo "ERROR: Cannot find nginx configurations. Is this an AppScale deployment?"
+        exit 1
+fi
+
+VIEWER_CONF=/etc/nginx/sites-enabled/appscale-${APP_ID}_datastore_viewer.conf
+if [ -e "$VIEWER_CONF" ]; then
+        VIEWER_PORT=$(grep listen ${VIEWER_CONF} | sed -r "s/.*listen ([0-9]+).*/\1/")
+        rm ${VIEWER_CONF}
+        service nginx reload
```

Comment:
You might want to add an `nginx -t` here just in case the user has some custom invalid nginx configuration. I would also recommend adding a `set -e` and `set -u` to the top of new bash scripts when possible. They provide an additional safeguard when refactoring and help prevent unexpected behavior if things go wrong.